packages/wrangler/src/__tests__/helpers/mock-kv.ts

```diff
@@ -4,13 +4,14 @@ export function mockKeyListRequest(
   expectedNamespaceId: string,
   expectedKeys: string[],
   keysPerRequest = 1000,
-  blankCursorValue: "" | undefined | null = undefined
+  blankCursorValue: "" | undefined | null = undefined,
+  expiration?: number | null
 ) {
   const requests = { count: 0 };
   // See https://api.cloudflare.com/#workers-kv-namespace-list-a-namespace-s-keys
   const expectedKeyObjects = expectedKeys.map((name) => ({
     name,
-    expiration: 123456789,
+    ...(expiration !== null && { expiration: expiration || 123456789 }),
```

Comment:
I think we should bit the bullet and change this helper so that by default we don't add an expiration. No tests are specifically checking that the expiration value is being set, other than updating a few snapshots.
We could just add a single test that makes sure that when an expiration is set then it appears in the upload. Otherwise just default to no-expiration.

This would make this helper simpler and also make the usage a bit less awkward where we are having to pass `null` to avoid sending this field.

Reply:
Yeah, you're right. Alright, lemme fix the helper and redo these tests. Might take a day or so.

Reply:
done